gs_chunked_io/writer.py

```diff
@@ -80,19 +97,28 @@ def close(self):
             if self._buffer:
                 self._put_part(self._current_part_number, self._buffer)
             self._compose_dest_blob()
+            if self.executor:
+                self.executor.shutdown()
 
     def _compose_dest_blob(self):
+        if self.executor:
```

Comment:
Should these three lines just be `self.wait()`?

Reply:
will fix